applications/conf/routes

```diff
@@ -43,8 +43,7 @@ GET        /index/contributors
 GET        /index/contributors/*contributor                                     controllers.TagIndexController.contributor(contributor)
 
 GET        /embed/video/*path                                                   controllers.EmbedController.render(path)
-GET        /embed/atom/media/:id                                                controllers.MediaAtomEmbedController.render(id)
-GET        /embed/atom/story-questions/:id                                      controllers.StoryQuestionsAtomEmbedController.render(id)
+GET        /embed/atom/$atomType<media|storyquestions>/:id                      controllers.AtomPageController.render(atomType: String, id: String)
```

Comment:
I don't think we should put directly `media` or `storyquestions`, but rather:
`/embed/atom/:atomType/:id`